charts/kubedb-catalog/templates/mongodb/percona-3.6.18.yaml

```diff
@@ -0,0 +1,25 @@
+{{ if .Values.catalog.mongo }}
+{{ if not .Values.skipDeprecated }}
```

Comment:
This check should not be done for non deprecated Versions

Reply:
fixed